hack/lib/common.sh

```diff
@@ -120,6 +120,14 @@ function build_sdk {
   mv ./build/operator-sdk "$1"/bin/operator-sdk
 }
 
+# Add kustomize to env
+function setup_kustomize {
+  header_text "getting kustomize"
+  curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+  chmod u+x kustomize
+  mv kustomize "$1"/bin/kustomize
+}
```

Comment:
Do we need this? I figured that since the `make` targets that require `kustomize` depend on it, and then the `kustomize` target would download it if it isn't already on the path.

Also it looks like the `install_kustomize.sh` script downloads the latest version of kustomize, whereas the Makefile has a hardcoded version. Shouldn't we be testing with the version that is scaffolded in the Makefile?